tests/test_inputmedia.py

```diff
@@ -583,6 +677,38 @@ async def test_send_media_group_default_protect_content(
         )
         assert not all(msg.has_protected_content for msg in unprotected)
 
+    @flaky(3, 1)
+    @pytest.mark.parametrize("default_bot", [{"parse_mode": ParseMode.HTML}], indirect=True)
+    async def test_send_media_group_default_parse_mode(
```

Comment:
please also test with explicitly passing `parse_mode=None` - in that case the resutling caption should read `"<b>photo</b> 1"` and there should be no entities

Reply:
Yep, done in b23acc53531dc4ec3f35bea791393d62bd01525d.